src/components/TimePicker/TimePicker.jsx

```diff
@@ -226,20 +226,9 @@ class TimePicker extends React.Component<Props, State> {
 
     const currentOption = this.state.options.find(item => item.value === currentValue);
 
-    const restProps = omit(
-      rest,
-      'defaultOption',
-      'hidden', // TODO(grozki): Get this to work?
-      'maxTime',
-      'minTime',
-      'timeFormat',
-      'timeIntervalMinutes',
-      'title' // TODO(grozki): Get this to work?
-    );
```

Comment:
I'm not sure you want to pass those, since React-Select doesn't handle those.
Might be worth to still omit them.